pkg/routeagent_driver/cabledriver/xfrm.go

```diff
@@ -15,33 +15,42 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package cleanup
+package cabledriver
 
 import (
 	"fmt"
 	"syscall"
 
+	"github.com/submariner-io/submariner/pkg/event"
+
 	"github.com/submariner-io/admiral/pkg/log"
 	"github.com/submariner-io/submariner/pkg/netlink"
 	"k8s.io/klog"
-
-	"github.com/submariner-io/submariner/pkg/routeagent_driver/cleanup"
 )
 
-type XFRMCleanupHandler struct {
+type Handler struct {
+	event.HandlerBase
 	netLink netlink.Interface
 }
 
-func NewXFRMCleanupHandler() cleanup.Handler {
-	return &XFRMCleanupHandler{netLink: netlink.New()}
+func NewXfrm() *Handler {
+	return &Handler{netLink: netlink.New()}
 }
 
-func (xc *XFRMCleanupHandler) GetName() string {
-	return "XFRM cleanup handler"
+func (xfrm *Handler) GetHandlerType() event.HandlerType {
```

Comment:
It's recommended to keep receiver vars to 1 or 2 chars.

```suggestion
func (h *Handler) GetHandlerType() event.HandlerType {
```

Reply:
done.